share/functions/__fish_print_function_prototypes.fish

```diff
@@ -1,5 +1,5 @@
 
 function __fish_print_function_prototypes -d "Prints the names of all function prototypes found in the headers in the current directory"
-    cat *.h* | sed -n "s/^\(.*[^[a-zA-Z_0-9]\|\)\([a-zA-Z_][a-zA-Z_0-9]*\) *(.*);.*\$/\2/p"
+    sed -n "s/^\(.*[^[a-zA-Z_0-9]\|\)\([a-zA-Z_][a-zA-Z_0-9]*\) *(.*);.*\$/\2/p" *.h*
```

Comment:
Then that would be a bug - you should be seeing "No matches for wildcard '*definitlynomatch*'". Or you are secretly using fish 2.2.0.